sources/packages/backend/libs/services/src/notifications/notification/notification.model.ts

```diff
@@ -26,6 +26,15 @@ export interface MinistryStudentFileUploadNotification {
   userId: number;
 }
 
+export interface PartialStudentMatchNotification {
+  givenNames: string;
+  lastName: string;
+  dob: Date;
```

Comment:
We should follow the other personalization in the same file and declare `dob` as `birthDate`. 
The business requirement has it defined as `dob` but we do not need to reuse the same variable name.

Reply:
updated